Changelog.md

```diff
@@ -1,9 +1,17 @@
 # Motoko compiler changelog
 
+## 0.10.0 (2023-09-08)
```

Comment:
Do we want to mention the generated Candid doc comments (https://github.com/dfinity/motoko/pull/4178)? This might be considered a breaking change for developers relying on the original generated output.

Reply:
We should note it if it has external relevance! I was thinking that "a doc comment is just a comment" and didn't really consider it. Feel free to amend the PR with a commit to `Changelog.md`.